docs/index.rst

```diff
@@ -3,50 +3,204 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-==================
-pAPRika
-==================
 
-pAPRika is a python toolkit for setting up, running, and analyzing free-energy molecular dynamics simulations.
 
-========
+.. role:: green-font
+.. role:: red-font
+.. role:: ignore-width
 
-Index
------
+.. |tick|    replace:: :green-font:`✓`
+.. |cross|   replace:: :red-font:`✕`
+.. |delta|   unicode:: U+0394
+.. |ast|     replace:: :ignore-width:`*`
+.. |dast|    replace:: :ignore-width:`**`
+.. |br| raw:: html
 
+   <br />
+
+.. image :: _static/images/paprika.png
+   :width: 350px
+   :align: left
+
+|
+|
+|
+|
+|
+|
+
+*pAPRika is a python toolkit for setting up, running, and analyzing free-energy molecular dynamics simulations.*
+
+
+===================
+
+
+Theory
+******
+
+
+Binding free-energy
+===================
+
+In the attach-pull-release (APR) method, a guest molecule is physically pulled out of the host molecule in a defined
+path by applying harmonic restraints. The binding free-energy is then computed in terms of the sum of the work required
+for (1) `attaching` restraints to the bound complex, (2) `pulling` the guest molecule out of the host molecule and (3)
+`releasing` the restraints of the unbound complex:
+
+.. math ::
+   \Delta G^\circ_\text{bind} = -(W_\text{attach} + W_\text{pull} + W_\text{release-conf} + W_\text{release-std})
+   :label: eq:bind
+
+
+.. image :: _static/images/pmf.png
+   :width: 600px
+   :align: center
+
+In the equation above, the work of releasing the restraints is split into two: :math:`W_\text{release-conf}` and
+:math:`W_\text{release-std}`. These represents the work for releasing conformational restraints (applied to host or
+guest molecules) and the guest's translational and rotational restraints to the standard concentration, respectively.
+The translational and rotational degrees of freedom of the guest molecule is defined in spherical coordinates
+:math:`(r,\theta,\phi)` and Euler angles :math:`(\alpha,\beta,\gamma)`, respectively. Unlike the work for releasing
+conformational restraints, the work for releasing the guest's restraints to the standard concentration can be evaluated
+analytically:
+
+.. math ::
+   \begin{eqnarray}
+      W_\text{release-std} & = & RT\,\text{ln}\left( \frac{C^{\circ}}{8\pi^2} \right) \\
+         & + & RT\,\text{ln}\left(\int^{\infty}_{0}\int^{\pi}_{0}\int^{2\pi}_{0} e^{-\beta U(r,\theta,\phi)} r^2 \text{sin}(\theta) \, d\phi d\theta dr \right) \\
```

Comment:
I think `\,` between the derivative `d` too.

Reply:
Yeah, it looks better with a space between the derivatives.